### PR TITLE
Corrected files after feedback

### PR DESCRIPTION
**Pull request by dianapvalles**
- Created immutable list // Added dependency for this
- Picked a random element from immutable list and passed it over as a string
- Closed img tags
- Changed ambiguous names of ID and function
- Changed the fetch response to receive text
- Modify try-catch

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -13,7 +13,11 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
-
+import com.google.common.collect.ImmutableList; 
+import java.util.*;
```
Comment (wait321) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Don't import *. It makes it harder to figure out dependencies.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -23,10 +27,11 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private static final ImmutableList<String> immutableList = ImmutableList.of("Mango","Strawberry","Banana");
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("<p>Favorite fruit:</p>" + immutableList.get(0));
```
Comment (wait321) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: pick a random one from the list. Not just the first one every time.
https://www.geeksforgeeks.org/generating-random-numbers-in-java/

File: walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html [hunk 3]
```diff
@@ -10,7 +10,7 @@
     <div id="content">
       <h1>Random Quotes</h1>
       <p>Click here to see a random quote:</p>
-      <button onclick="getRandomQuote()">Fetch a quote!</button>
+      <button onclick="getRandomQuoteUsingAsyncAwait()">Fetch a quote!</button>
```
Comment (wait321) [inline walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html]: looks like you were playing around with the example. revert for the commit.

**Review by wait321 — CHANGES_REQUESTED** [review]
Minor changes. Great job overall.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (dianapvalles) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Sound good, thank you.

**Pull request closed without merging**